content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -32,7 +32,7 @@ similar to a branch.
 
 </details>
 
-Like with regular experiments, checkpoints can become persistent by
+Like with regular experiments, checkpoints can be persisted by
 [committing them to Git](#committing-checkpoints-to-git).
```

Comment:
Yeah we discussed this term and at some point decided not to use it as a verb since it's not really a the actual meaning of "to persist" in general language.

We do still have this problem in a couple places: in the title of https://dvc.org/doc/user-guide/experiment-management/persisting-experiments and in a hidden section under https://dvc.org/doc/user-guide/experiment-management/running-experiments#the-experiments-queue.

Full discussion: https://github.com/iterative/dvc.org/issues/2960

Reply:
@jorgeorpinel  - Thanks for bringing this discussion to my attention.
I was not aware of the rabbit hole 🐇  here 😅 . My edit here was strictly related to the usage as a verb (english) - my 2 cents about that (coincidentally 2 points 😄  ):

1. re: [first point raised in the issue description](https://github.com/iterative/dvc.org/issues/2960) - Though _persist_ also means "continuing to do something in a determined way" the usage here of "continuing to exist (maybe past an expected/usual time)" is definitely _also_ a valid definition in the general language (See: [[1]](https://dictionary.cambridge.org/dictionary/english/persist) [[2]](https://www.merriam-webster.com/dictionary/persist) [[3]](https://www.oxfordlearnersdictionaries.com/definition/american_english/persist) for example).
2. Probably more important - This is how it is conventionally used in our domain. "To persist X" means "to commit X to persistency" (of some sorts). So, the way I see it (and the reason I bothered "fixing" it) "checkpoints can become persistent by..." - would sound wrong / clunky to software engineers & people in associated fields (e.g. data engineering) - we have a verb for that. I would argue that familiarity and conciseness win here (even if it wasn't for my point no. 1)

But, on a more constructive note - As per the [last comment](https://github.com/iterative/dvc.org/issues/2960#issuecomment-999627440) am I correct to assume it's agreed if I make this change (only here): `Like with regular experiments, checkpoints can become persistent by committing them to git` -> `Like with regular experiments, checkpoints can be committed to git` [losing info here, not loving it, but better than now]

I was actually going to suggest following your comment to change the other places you mentioned - namely the section title from `Persisting experiments` to `Committing experiments` - but upon further consideration I actually find that problematic 😢 . IMHO "committing" is more of an implementation detail here. If I were to read/skim these docs and compare to other experiment tracking solutions I would look for (search / grep) the terms "persist" and "track" (=what) and not "commit" (=how) - all the more important when it comes to titles!

Reply:
> "continuing to exist (maybe past an expected/usual time)" is definitely also a valid definition

Yes but then the phrase should be "checkpoints can persist" which would be unclear. "Become persistent" is a little clunky indeed, but clear: going from temporary to continuous.

Anyway I see you changed it now along the lines of the decisions in #2960 so 👍🏼  

If you want to argue for another phrasing please comment in the issue, we can always revisit all these mentions.